WeakAuras/Prototypes.lua

```diff
@@ -6379,10 +6388,10 @@ Private.event_prototypes = {
       },
       {
         name = "legendaryIcon",
-        display = L["Use Legendary Power Icon"],
+        display = L["Fetch Legendary Power"],
         type = "toggle",
         test = "true",
-        desc = L["Displays the icon for the Legendary Power that matches this bonus id."],
+        desc = L["Fetches the name and icon of the Legendary Power that matches this bonus id."],
```

Comment:
I checked all the modernize code and there was only one instance of renaming variables and this change did not fit that scenario so I left the variable name alone instead of changing it and modernizing.

Reply:
Yeah it's usually not worth doing that. Because:
* We (devs/alpha testster) don't usually test the modernize code as well as the rest.
* The cost of a mistake in it are much higher. There's always the potential to make mistakes.